### PR TITLE
Add patches for CVEs 2022-34749 and 2020-15123

### PR DESCRIPTION
**Pull request by msadang**
{# Patch for CVE-2022-34749 https://github.com/advisories/GHSA-fw3v-x4f2-v673 #}
{# Patch for CVE-2020-15123 https://nvd.nist.gov/vuln/detail/CVE-2020-15123 #}

### CODE REVIEW

File: templates/rapidsai-core/partials/patch.dockerfile.j2 [hunk 1]
```diff
@@ -1,8 +1,12 @@
 {# This partial is used to install patches to resolve known CVEs #}
 
 {# Patch for CVE-2020-8116 https://github.com/advisories/GHSA-ff7x-qrg7-qggm #}
+{# Patch for CVE-2022-34749 https://github.com/advisories/GHSA-fw3v-x4f2-v673 #}
+{# Patch for CVE-2020-15123 https://nvd.nist.gov/vuln/detail/CVE-2020-15123 #}
 RUN source activate rapids \
     && npm i -g npm@">=7.0"
+    && npm i -g codecov@">=3.7.1"
+    && conda install mistune=2.0.3
```
Comment (raydouglass) [inline templates/rapidsai-core/partials/patch.dockerfile.j2:9]: Should make this >=2.0.3 to account for future versions.

**Review by raydouglass — CHANGES_REQUESTED** [review]

File: templates/rapidsai-core/partials/patch.dockerfile.j2 (hunk 1 above)
Comment (msadang) [reply templates/rapidsai-core/partials/patch.dockerfile.j2:9]: Will make the change

File: templates/rapidsai-core/partials/patch.dockerfile.j2 [hunk 2]
```diff
@@ -1,8 +1,12 @@
 {# This partial is used to install patches to resolve known CVEs #}
 
 {# Patch for CVE-2020-8116 https://github.com/advisories/GHSA-ff7x-qrg7-qggm #}
+{# Patch for CVE-2022-34749 https://github.com/advisories/GHSA-fw3v-x4f2-v673 #}
+{# Patch for CVE-2020-15123 https://nvd.nist.gov/vuln/detail/CVE-2020-15123 #}
 RUN source activate rapids \
     && npm i -g npm@">=7.0"
+    && npm i -g codecov@">=3.7.1"
+    && conda install mistune>=2.0.3
```
Comment (raydouglass) [inline templates/rapidsai-core/partials/patch.dockerfile.j2:9]:
```suggestion
    && conda install "mistune>=2.0.3"
```

File: templates/rapidsai-core/partials/patch.dockerfile.j2 [hunk 3]
```diff
@@ -1,8 +1,12 @@
 {# This partial is used to install patches to resolve known CVEs #}
 
 {# Patch for CVE-2020-8116 https://github.com/advisories/GHSA-ff7x-qrg7-qggm #}
+{# Patch for CVE-2022-34749 https://github.com/advisories/GHSA-fw3v-x4f2-v673 #}
+{# Patch for CVE-2020-15123 https://nvd.nist.gov/vuln/detail/CVE-2020-15123 #}
 RUN source activate rapids \
     && npm i -g npm@">=7.0"
+    && npm i -g codecov@">=3.7.1"
+    && conda install "mistune>=2.0.3"
```
Comment (raydouglass) [inline templates/rapidsai-core/partials/patch.dockerfile.j2:9]:
```suggestion

RUN gpuci_mamba_retry install -y -n rapids "mistune>=2.0.3"
```
Should use mamba here instead. And run it outside of the `source activate rapids`.

File: templates/rapidsai-core/partials/patch.dockerfile.j2 [hunk 4]
```diff
@@ -1,8 +1,13 @@
 {# This partial is used to install patches to resolve known CVEs #}
 
 {# Patch for CVE-2020-8116 https://github.com/advisories/GHSA-ff7x-qrg7-qggm #}
+{# Patch for CVE-2022-34749 https://github.com/advisories/GHSA-fw3v-x4f2-v673 #}
+{# Patch for CVE-2020-15123 https://nvd.nist.gov/vuln/detail/CVE-2020-15123 #}
 RUN source activate rapids \
     && npm i -g npm@">=7.0"
```
Comment (raydouglass) [inline templates/rapidsai-core/partials/patch.dockerfile.j2:7]:
```suggestion
    && npm i -g npm@">=7.0" \
```
Need the trailing slash

**Review by raydouglass — APPROVED** [review]

**Pull request merged**